tox.ini

```diff
@@ -1,12 +1,23 @@
 [tox]
 skipsdist = True
-envlist = py37-lint, py37-mypy
+envlist = py37-{lint,mypy}, docs
 
 [testenv]
 deps =
-  py37-lint: pylint
-  py37-mypy: mypy
+  lint: pylint~=2.3.1
+  mypy: mypy~=0.711
+
+
+commands =
+  lint: pylint opentelemetry-api/opentelemetry/
+  mypy: mypy opentelemetry-api/opentelemetry/
+
+[testenv:docs]
```

Comment:
Instead of having two settings for testenv (unless you want to run tox -e ...), you can take advantage of tox's conditional settings to make the file more brief. 

![image](https://user-images.githubusercontent.com/11580155/60772751-a9342a80-a0af-11e9-99fb-51da12931bbc.png)

Reply:
But would it be shorter? In this case I thought putting it in an en extra environment would look nicer (since there are few things in common with other environments, and I guess we also don't want docs to execute for all different python versions).